protobuf/flow/execution/execution.proto

```diff
@@ -19,15 +19,17 @@ service ExecutionAPI {
     // Scripts
 
     // ExecuteScriptAtBlockID executes a ready-only Cadence script against the execution state at the block with the given ID.
-    rpc ExecuteScriptAtBlockID (ExecuteScriptAtBlockIDRequest) returns (ExecuteScriptResponse);
+    rpc ExecuteScriptAtBlockID (ExecuteScriptAtBlockIDRequest) returns (ExecuteScriptAtBlockIDResponse);
```

Comment:
There is only one Script related API unlike the access api which has several. Hence, making the response name more specific.